commands/namespaces/list.go

```diff
@@ -0,0 +1,122 @@
+// everest
+// Copyright (C) 2025 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package namespaces provides the namespaces CLI command.
+package namespaces
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+
+	"github.com/percona/everest/pkg/cli"
+	"github.com/percona/everest/pkg/cli/namespaces"
+	"github.com/percona/everest/pkg/logger"
+	"github.com/percona/everest/pkg/output"
+)
+
+var (
+	namespacesListCmd = &cobra.Command{
+		Use:     "list [flags]",
+		Args:    cobra.NoArgs,
+		Long:    "List an existing namespaces",
+		Short:   "List an existing namespaces",
```

Comment:
the description doesn't seem to be grammatically correct, please consider any of the options below: 
- "List existing namespaces"
- "List all existing namespaces"
- "List the existing namespaces"

Reply:
done